app/src/test/java/quotes/AppTest.java

```diff
@@ -4,11 +4,23 @@
 package quotes;
 
 import org.junit.jupiter.api.Test;
+
+import java.io.File;
+
 import static org.junit.jupiter.api.Assertions.*;
 
 class AppTest {
-    @Test void getRandomQuote() {
+    @Test void ensureRunning() {
         App app = new App();
         assertNotNull(app);
     }
+
+    @Test public void testFetchRandomQuote() {
+        App app = new App();
+        File quoteFile = new File("../app/src/main/resources/recentquotes.json");
+        assertTrue(quoteFile.exists());
+
+    }
+
```

Comment:
For the rest of your tests, look back at the feature tasks for the lab:
- "When running your application...make a request to an API" - can you test to see if the connection is successful?
- "Ensure that if your app has errors in connecting to the Internet, you instead display a random quote from your file" - you could write a test with a deliberately bad URL (or something else that would cause the connection to fail) and make sure something still comes back

Also think about the feature tasks from the previous lab, and see if you can write some more specific tests for those.